tests/4.5/target_teams_distribute/test_target_teams_distribute_default_none.F90

```diff
@@ -0,0 +1,90 @@
+!===--- test_target_teams_distribute_default_none.F90-----------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This tests uses the default(none) clause on a target teams distribute test.
+! The test aims to validate that all values will not have default data sharing
+! attributes.
+!
+!===------------------------------------------------------------------------===//
+
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_teams_distribute_default_none
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  CHARACTER(len=300) :: infoMessage
+  WRITE(infoMessage, *) "Test only uses the default(none) clause and &
+       & does not guarantee that the default(none) is enforced."
+  OMPVV_WARNING(infoMessage)
+
+  OMPVV_TEST_VERBOSE(default_none1() .ne. 0)
```

Comment:
Add comment what the tests are checking.

Reply:
Added short comments to each